### PR TITLE
update Capacity, Ward, to move occupancy to Ward

### PR DESCRIPTION
**Pull request by jeraldkiew**

### CODE REVIEW

File: src/main/java/seedu/address/model/ward/Ward.java [hunk 1]
```diff
@@ -48,18 +51,36 @@ public static boolean isValidWard(Name name) {
...
     public Capacity getCapacity() {
         return capacity;
     }
+
+    public int getOccupancy() {
+        return patients.size();
+    }
+
+    public boolean isFull() {
+        return getOccupancy() == capacity.getValue();
```
Comment (daytona65) [inline src/main/java/seedu/address/model/ward/Ward.java:75]:
```
suggestion
return getOccupancy() >= capacity.getValue()
```

File: src/main/java/seedu/address/model/ward/Ward.java [hunk 2]
```diff
@@ -48,18 +51,36 @@ public static boolean isValidWard(Name name) {
         return name.toString().matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Returns true if a given list of patients is small enough to fit the
+     * ward's capacity
+     */
+    private boolean isValidPatients(List<Patient> patients2) {
+        return patients2.size() <= capacity.getValue();
+    }
```
Comment (daytona65) [inline src/main/java/seedu/address/model/ward/Ward.java:60]: Maybe check also for unique list of patients?

Comment (jeraldkiew) [reply src/main/java/seedu/address/model/ward/Ward.java:60]: unique-ness checked in UniquePatientList

**Review by yitong241 — APPROVED** [review]
Looks good to merge.

**Pull request merged**